tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -0,0 +1,91 @@
+"""Schema for HEADER row of all submission types."""
+
+
+from ...util import MultiRecordRowSchema, RowSchema, Field
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T3
+
+
+child_one = RowSchema(
+    model=TANF_T3,
```

Comment:
@elipe17 see [link](https://gorafttech.sharepoint.com/:u:/r/sites/TDRSResearchDesign/Shared%20Documents/Dev/dev/Parsing/py/[TANF_Parsing.py](https://gorafttech.sharepoint.com/:u:/r/sites/TDRSResearchDesign/Shared%20Documents/Dev/dev/Parsing/py/TANF_Parsing.py?csf=1&web=1&e=qFRqo9)?csf=1&web=1&e=qFRqo9) to T3 record field names.